.gitignore

```diff
@@ -24,4 +24,7 @@ project.lock.json
 *DS_Store
 *.ncrunchsolution
 *.*sdf
-*.ipch
\ No newline at end of file
+*.ipch
+.vs
+.build/
+.testPublish
```

Comment:
will do (although it shouldn't matter too much since there are no tests in this repo). btw. it also means that the repo I copied this from (don't remember which one) is missing the trailing slash as well...